app/controllers/favorites_controller.rb

```diff
@@ -0,0 +1,36 @@
+# responsible for creating and deleting user's favorite tracks
+# depends on asset_id
+# user must be logged_in
+# when favorite track is created it triggers favorites_count counter cache on asset
```

Comment:
Can we remove most of these comments? Some of them seem to not add anything / duplicate the code.